safe_transaction_service/history/models.py

```diff
@@ -1806,6 +1806,15 @@ def get_delegates_for_safe_and_owners(
             .distinct()
         )
 
+    def remove_delegates_for_owner_in_safe(
+        self, safe_address: ChecksumAddress, owner_address: ChecksumAddress
+    ) -> int:
+        return (
+            self.filter(Q(safe_contract_id=safe_address))
+            .filter(delegator=owner_address)
```

Comment:
No need for `Q`, also you can do the 2 queries together

```suggestion
            self.filter(safe_contract_id=safe_address, delegator=owner_address)
```

Reply:
Fast copy/paste, sorry for that 😓